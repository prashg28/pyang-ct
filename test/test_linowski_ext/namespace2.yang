 module namespace2 {

   namespace "urn:org:example:xml:ct:ns2";
   prefix "ns2";

   import complex-types {
       prefix "ct"; 
   }

   import namespace1 {
       prefix ns1;
   }

   ct:complex-type complex {
       ct:extends ns1:base;
       leaf base-leaf {
           type int8;
       }
   }

   ct:complex-type complex2 {
       ct:extends complex;
       leaf base-leaf {
           type int8;
       }
   }

   ct:instance complex {
       ct:instance-type complex;
   }

}
