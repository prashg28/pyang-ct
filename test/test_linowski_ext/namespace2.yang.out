namespace2.yang:22: error: DUPLICATE_CHILD_NAME
